test/org/sagebionetworks/bridge/services/UploadServiceTest.java

```diff
@@ -1,17 +1,10 @@
 package org.sagebionetworks.bridge.services;
 
-import static com.amazonaws.services.s3.model.ObjectMetadata.AES_256_SERVER_SIDE_ENCRYPTION;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
-import java.io.IOException;
-import java.net.URISyntaxException;
-import java.util.ArrayList;
-import java.util.List;
-
-import javax.annotation.Resource;
-
+import com.amazonaws.AmazonClientException;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.ObjectMetadata;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import org.apache.commons.codec.binary.Base64;
 import org.apache.commons.codec.digest.DigestUtils;
 import org.apache.http.client.methods.CloseableHttpResponse;
```

Comment:
You're getting a lot of import statement churn, we should check how you're sorting imports and see if we can get it to be similar to what Dwayne and I have set in our IDEs. Just stop by tomorrow and we'll try and prevent this. Makes the diffs easier to review.